### DISCUSSION TITLE
作为一个账号的主机系统，我这边要重装系统，之后登录会有问题吗

### DISCUSSION THREAD

**Discussion by GithubLexi666** [getlantern/lantern-forum-cn#654] [Q&A]
你好，我这边是以我的电脑注册的蓝灯这个账号，目前电脑打算重装系统，重装之后会影响登录吗？很急，因为约了师傅明天就装 @ @Mike777Khoo X谢谢

**Comment by MaxMKMax** [comment]
@GithubLexi666 您好. 只要您的系统是新版而不是老旧版本的话是没问题的. 入股您遇到问题, 可以参考这个链接:
https://lantern.io/zh-Hans/faq#use
